core/src/test/scala/org/broadinstitute/dsde/rawls/bucketMigration/BucketMigrationServiceSpec.scala

```diff
@@ -248,6 +256,42 @@ class BucketMigrationServiceSpec extends AnyFlatSpec with TestDriverComponent {
     exception.errorReport.statusCode shouldBe Some(StatusCodes.BadRequest)
   }
 
+  it should "fail for non-US multiregion buckets" in withMinimalTestDatabase { _ =>
+    val adminService = mockBucketMigrationServiceForAdminUser()
+
+    val nonUsWorkspace = minimalTestData.workspace.copy(name = "nonUsWorkspace",
```

Comment:
Nit: I would be more comfortable if the bucketName was one of the actual single region names that we expect… just to be very clear that we don't match on it.

Reply:
Agreed! This is just the bucket name though, not the region. The region we're checking is returned by the mocked `Bucket` down on [l. 277](https://github.com/broadinstitute/rawls/pull/2505/files/13393d1253edefffd14aa4311fc0cea74cd75063#diff-cfab8b4fd2e70ce3dea44657883d0898a691834bca3aff618c351909d8576b9cR277) which is one of the regions we expose in the UI (`northamerica-northeast1`)

Reply:
Ah, I was confusing bucketName and region. Thanks!